app/models/form_datetime_field_value.rb

```diff
@@ -2,11 +2,12 @@
 
 class FormDatetimeFieldValue < FormFieldValue
   def value=(val)
-    if val.is_a?(Array)
-      self.datetime = val[0]
-      self.datetime_to = val[1]
+    parse_val = JSON.parse(val || '[]')
```

Comment:
add error handling for parsing.
Also parsing predefined value doesn't look good